eva-accession-import/src/test/java/uk/ac/ebi/eva/accession/dbsnp/processors/SubSnpNoHgvsToSubmittedVariantProcessorTest.java

```diff
@@ -420,6 +420,48 @@ public void transformShortTandemRepeat() throws Exception {
                                false);
     }
 
+    @Test
+    public void transformShortTandemRepeatReverseStrand() throws Exception {
```

Comment:
Do these methods throw exceptions?

Reply:
not really, but the 'process' method declares it throws, so you have to catch it or throw them.